modules/logging/ui/src/sap/commerce/toolset/logging/ui/tree/nodes/RemoteHacInstancesLoggersOptionsNode.kt

```diff
@@ -20,11 +20,12 @@ package sap.commerce.toolset.logging.ui.tree.nodes
 
 import com.intellij.openapi.project.Project
 import sap.commerce.toolset.HybrisIcons
+import sap.commerce.toolset.hac.exec.HacExecConnectionService
 
 class RemoteHacInstancesLoggersOptionsNode(project: Project) : LoggersOptionsNode("Remote hAC Instances", HybrisIcons.Y.REMOTES, project) {
 
     override fun getNewChildren(nodeParameters: LoggersNodeParameters): Map<String, LoggersHacConnectionNode> = nodeParameters.connections
-        .filter { it.value } // only active connections
-        .map { (connection, active) -> LoggersHacConnectionNode(connection, active, project) }
-        .associateBy { "${it.connectionSettings.uuid}_|_${it.activeConnection}" }
+        .filter { it == HacExecConnectionService.getInstance(project).activeConnection } // only active connections
```

Comment:
performance issue, extract to variable `HacExecConnectionService.getInstance(project).activeConnection`